src/PlayFabBuddy.Cli/README.md

```diff
@@ -23,6 +23,21 @@ Commands related to players.
     dotnet run -- players create <number of players>
 
 #### Delete
-**WARNING:** This will delete ALL Users from the Title!
+**WARNING:** This will delete **ALL** Master Player Accounts from the Title, which have been previously created with PlayFabBuddy on this very machine!
 
-    dotnet run -- players delete
\ No newline at end of file
+    dotnet run -- players delete
+
+#### Delete by Segment
+Will delete all Master Player Accounts from a given segment, identified by segment name.
+If a Master Player Account is assigned to another Title as well, players will **NOT** be deleted!
+
+    dotnet run -- players delete-by-segment <segment name>
+   
+As `segment name`, you may use one of the pre-defined segment names:
+
+* `All Players` - All players
+* `Lapsed Players` - Last login greater than 30 days ago
+* `Payers` - Total value to date in USD greater than $0.00
+* `Week Two Active Players` - First login less than 14 days ago AND First login greater than or equal to 7 days ago AND Last login less than 7 days ago
+
+> Please note: Due to the nature of how segments work, you may need to execute this action multiple times to make sure to really delete all players in the given segment.
```

Comment:
This reads like its behavior that I would not expect from the command, I would expect that it reliably would delete all players.

We should create a new issue to track this or fix it here.

Reply:
Agreed. I only discovered this when I was pretty done with the feature, and it seems to get less reliable during specific times of the day, potentially related to overall usage on PlayFab.

The question is: should we just retry until it does not find any more results?

Reply:
Let's work through this in #54